lib/knapsack_pro/formatters/fetch_time_tracker.rb

```diff
@@ -0,0 +1,12 @@
+module KnapsackPro
+  module Formatters
+    class FetchTimeTracker
```

Comment:
The following `TimeTrackerFetcher` sounds more like a service name than `FetchTimeTracker`. But feel free to discard it if you prefer your name.

```suggestion
    class TimeTrackerFetcher
```